cpp/ps0001/ps0001_sol.cpp

```diff
@@ -0,0 +1,16 @@
+#include <iostream>
+
+int main()
+{
+    // prompting user for 3 numbers to add 1.5/2
+    std::cout<<"Hello! please enter the numbers\n";
+    unsigned short a,b,c,sum;
+    std::cin>>a>>b>>c;
```

Comment:
- [ ] declared 3 variables to store user inputs with data type `unsigned short int`
- [x] used `std::cin` in a right way to prompt user for 3 numbers to add